docs/vgclog.md

```diff
@@ -0,0 +1,423 @@
+
+## Verbose GC log contents 
+
+The Verbose garbage collection logs are printed in XML format and consist of the following sections:  
+
+- a summary of your GC configuration, which is captured in the stanza beginning with the [`<INITIATIZED>` tag](vgclog.md#xml-tags-an-attributes) 
+
+- stanzas that contain information about the different garbage collection cycles run, including the GC increments and GC operations that made up the GC cycle. 
+
+The verbose GC logs are event-based, recording each *stop-the-world* event as it happens. While *stop-the-world* cycles are fully logged, some aspects of concurrent cycles are also logged, even though operations that are running concurrently with application threads are not logged. Concurrent operations run as part of a concurrent cycle, and these concurrent cycles begin and finish with *stop-the-world* events - events that are logged. (For some garbage collectors, the concurrent cycle also includes intermediate *stop-the-world* events). In addition, for some types of concurrent collections, verbose GC logs also include information about when certain targets are reached. 
+
+The highest level stanzas in the verbose GC log represent GC cycles and begin with xml tags that define the kickoff event and reason. Except for some of the more complex GC algorithms (such as the global marking phase GC of the balanced policy), a GC cycle, when complete, generally results in reclaimed memory for reuse. Each cycle consists of 1 or more GC increments and GC operations.  
+
+Events that make up a garbage collector cycle are listed in the stanza in a linear fashion, but are ‘nested’ by a start and end tag that identify the start and end of an event. Details of the event are recorded in the XML attributes associated with the tag and also within tags that are nested within the GC event's start and end tags.
+
+The following table shows the tagging that is used for the nested GC processes: 
+
+| GC Process   | Start and end tag in verbose GC log  | Details                                                         |
+|--------------|-------------------------------|-----------------------------------------------------------------|
+| GC cycle     | `<cycle-start>`, `<cycle-end>`| Completion of GC cycle results in reclaimed memory in most cases|
+| GC increment | `<gc-start>`, `<gc-end>`      | 1 or more GC increments make up a GC cycle                      |
+| GC operation | `<gc-op>`                     | 1 or more GC operations make up a GC increment. Examples of a GC operation include mark, sweep, scavenge.|
+
+**Note:** GC operations are the smallest fragment in a verbose GC log. All details of a GC operation are logged by using a single tag rather than start and end tags.  
```

Comment:
While it's a single tag, it will typically contain a few nested tags (object copied, soft ref processed etc).